src/pages/Login/index.css

```diff
@@ -18,3 +18,69 @@
 .call-to-action {
   margin-bottom: 3em;
 }
+
+.next-steps,
+.timeline {
+  display: flex;
+  justify-content: space-between;
+  align-items: start;
```

Comment:
this trigger a warning : (26:3) start value has mixed support, consider using flex-start instead

Reply:
same